tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -1292,6 +1292,9 @@ a dictionary can also be used with a named enum.
         # prints '[0, 5, 6]'
         print(State.values())
 
+If not assigning a value to a key of an enum it will be assigned the previous value plus one,
+or ``0`` if it is the first entry in the enum. Multiple keys with the same value are allowed.
+
```

Comment:
Is the double empty lines intended? Otherwise LGTM :)

Reply:
Yes kept the original double empty lines, might just have been because it was a code block above but was there originally